docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -245,16 +255,71 @@ slow to return results.
 (such as AWS Athena and BigQuery). Repeatedly querying for this data can easily
 rack up costs.
 
+## Optimizing Pre-Aggregation Build Times
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| For ideal performance, pre-aggregations should be built using a dedicated
+| Refresh Worker. [See here for more details][ref-prod-list-refresh].
+<!-- prettier-ignore-end -->
+
+By default, Cube.js will use the source database as a temporary staging area for
+writing pre-aggregations before moving them to Cube Store.
+
+> Get more info about default strategy
+
+### Batching
```

Comment:
Maybe streaming? Internally, we don't use batching as a definition/method name or etc, which will be a problem for users who will review our codebase.